go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/remind101/deploy
+
+go 1.16
+
+require (
+	bitbucket.org/kardianos/osext v0.0.0-20181027061946-15c52d0993e9 // indirect
+	github.com/BurntSushi/toml v0.1.1-0.20141028230038-3883ac1ce943 // indirect
+	github.com/bmizerany/assert v0.0.0-20120716205630-e17e99893cb6 // indirect
+	github.com/codegangsta/cli v1.6.0
+	github.com/fhs/go-netrc v1.0.0 // indirect
+	github.com/github/hub v2.11.2+incompatible
+	github.com/google/go-github v0.0.0-20150126182318-7ea4ee6d2226
```

Comment:
This is hella outdated, thoughts on upgrading in general? Also curious why this introduced breaking chances if it's supposed to be a direct migration to go modules without upgrades.

Reply:
I had to upgrade some things in order to get the build to work. Some of the transitive dependency packages no longer exist at their original locations.